doc/examples/fast_streamline_search.py

```diff
@@ -0,0 +1,203 @@
+"""
```

Comment:
Please also add it to `doc/example/valid_examples.txt`

Reply:
done